doc/nrf/libraries/emds/emds.rst

```diff
@@ -0,0 +1,52 @@
+.. _emds_readme:
+
+Emergency data storage
+######################
+
+.. contents::
+    :local:
+    :depth: 2
+
+The emergency data storage (EMDS) library provides functionality for storing of data with a guaranteed time, based on the amount of data that are being stored.
+It can be used in situations of critical power shortage to store data in memory before the battery that powers the device is about to fully discharge.
```

Comment:
```suggestion
The emergency data storage (EMDS) library provides functionality for storing of data within a guaranteed time, based on the amount of data that are being stored.
It can be used in situations of critical power shortage to store data in memory, e.g. before the battery that powers a device is depleted.
```